demos/html5/ckeditor4/package.json

```diff
@@ -5,10 +5,8 @@
   "description": "A simple html5 App integrating WIRIS MathType in a CKEditor4 rich text editor.",
   "main": "app.js",
   "scripts": {
-    "prestart-remote": "yarn && yarn unlink @wiris/mathtype-ckeditor4 && yarn unlink @wiris/mathtype-html-integration-devkit && yarn install --force",
-    "start-remote": "webpack-dev-server --open",
-    "prestart": "yarn && yarn link @wiris/mathtype-ckeditor4 && yarn link @wiris/mathtype-html-integration-devkit",
-    "start": "webpack-dev-server --open"
+    "start-remote": "yarn && yarn unlink @wiris/mathtype-ckeditor4 && yarn unlink @wiris/mathtype-html-integration-devkit && yarn install --force",
+    "start": "yarn && yarn link @wiris/mathtype-ckeditor4 && yarn link @wiris/mathtype-html-integration-devkit"
```

Comment:
Maybe we should rename these to `prestart` and `prestart-remote` as well to avoid confusion when reading the `package.json`?

Reply:
I updated the script names to `prestart` and `prestart-remote`.